### DISCUSSION TITLE
Parts of model become see-through in AR scene

### DISCUSSION THREAD

**Discussion by Jasonp8000** [google/model-viewer#2793] [Q&A]
Hi everyone,

I've noticed when viewing a model in the AR scene parts of the model become see-through. Has anyone else had this issue ?

![241199429_445364983450546_5687607005535723810_n](https://user-images.githubusercontent.com/90825510/133566765-b79d6b83-5c48-48d8-a5d6-a771b50d3919.jpg)

**Comment by elalish** [comment]
Is this in Scene Viewer mode? This is due to background occlusion and  it has erroneously found your scene's background to be closer than it really is (it's a tricky problem). I believe you can disable this feature in the top-right of the screen. You won't see this in WebXR mode as we don't support background occlusion  yet.

**Comment by SGECZ** [comment]
yeah - it's occlusion - you can switch it off, there's a button at the top

**Reply by Jasonp8000** [reply]
I'm new to using model viewer, which button an you referring to? Can occlusion automatically be turned off?

**Reply by SGECZ** [reply]
![Screenshot_20210923-121318_Google](https://user-images.githubusercontent.com/68609303/134490525-a8bdd021-43ae-4979-8765-05c809dc92f0.jpg)

and yes, it can

**Reply by Jasonp8000** [reply]
Is this different for iso devices?  I'm using an iPhone 12 pro. 
![242140912_1547178698949444_5999072454347928731_n](https://user-images.githubusercontent.com/90825510/134497930-56f4e34c-d944-4a8a-87c5-16e1c158bba8.jpg)

**Reply by elalish** [reply]
Yes, Scene Viewer is Android, while Quick Look is iOS. This is why it is always key to note your device, OS, and browser when asking a question. I don't think they have this option in Quick Look. Please file a bug on Apple.

**Comment by michaelgarydean** [comment]
Just to follow up on this - I noticed a similar effect seems to happen when faces are detected (at least on iOS / Quick Look). The model becomes see through to show the face. Is this the same setting, and only available on Android? I couldn't find an attribute in the model-viewer documentation to ignore facetracking.

**Reply by elalish** [reply]
Definitely not a MV setting, and  if it's on Quick Look, then it's not Android either. You'll have to check Apple's docs.
